frame/identity/src/lib.rs

```diff
@@ -201,6 +202,44 @@ pub mod pallet {
 		ValueQuery,
 	>;
 
+	#[pallet::genesis_config]
+	pub struct GenesisConfig<T: Config> {
+		pub identities: Vec<(T::AccountId, String)>,
```

Comment:
I think you want this
```suggestion
		pub identities: Vec<(T::AccountId, IdentityInfo)>,
```

Reply:
Even so, it seems much more preferable to be able to specify all other identity-related fields in genesis config, so let's figure out what the trait bounds should be here...

Oh, it's just `T::MaxAdditionalFields`, so we should just use that:

```suggestion
		pub identities: Vec<(T::AccountId, IdentityInfo<T::MaxAdditionalFields>)>,
```

Reply:
Thanks for comments both, I didn't use `IdentityInfo` initially as it wanted `Deserialize<'_>`, I agree it is preferable to support this though, I will try to add `serde::[Des|S]erialize` on these types

Reply:
That is what i meant with trait bounds. Normally we just use POD types here to not having to use Serde from within the runtime.  
I think its fine.

Reply:
A `Vec<u8>` would be a bit less biased than a String - otherwise the encoding is not fixed.

Reply:
We'll probably need `BoundedVec` then, no reason to not make this derive MEL.

Reply:
thx, uses `BoundedVec` now